docs/howto/configure/pangeo-requester-pays-access.md

```diff
@@ -0,0 +1,103 @@
+# Pangeo Data Access via Requester Pays
```

Comment:
I think we could use a top-level section like `Data access` and then this could be the first content in that section. I think that could bring this in-line with our Hub Administrator's guide "topic guides" here: https://pilot.2i2c.org/en/latest/